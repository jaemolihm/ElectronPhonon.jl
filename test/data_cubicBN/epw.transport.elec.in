--
&inputepw
  prefix      = 'bn'
  amass(1)    = 10.811
  amass(2)    = 14.0067
  outdir      = 'temp/'
  dvscf_dir   = './save/'
  etf_mem     = 0

  elph        = .true.
  epbread     = .false.
  epwread     = .true.

  lpolar      = .true.
  lifc        = .false.
  use_ws      = .true.
  vme         = 'wannier'

  scattering  = .true.
  iterative_bte = .true.
  mob_maxiter = 30
  broyden_beta= 1.0
  int_mob     = .false.
  carrier     = .true.
  ncarrier    = 1E20
  mp_mesh_k   = .true.

  efermi_read = .true.
  fermi_energy= 15.4 ! eV
  fsthick     =  0.4 ! eV

  ! Valence band maximum    =  10.921574 eV
  ! Conduction band minimum =  15.373135 eV

  degaussw    = 0.08 ! eV
  nstemp      = 2
  temps       = 200.0 300.0 400.0 ! K

  nkf1        = 15
  nkf2        = 15
  nkf3        = 15

  nqf1        = 15
  nqf2        = 15
  nqf3        = 15

!  use_opt_w2b = .true.

  nbndsub     =  8

  wannierize  = .false.
  num_iter    = 1000
  dis_win_max = 36.0
  dis_froz_max= 25.0

  nk1         = 4
  nk2         = 4
  nk3         = 4

  nq1         = 2
  nq2         = 2
  nq3         = 2
 /
