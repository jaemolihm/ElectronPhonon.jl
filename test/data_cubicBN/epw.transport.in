--
&inputepw
  prefix      = 'bn'
  amass(1)    = 28.086
  outdir      = 'temp/'
  dvscf_dir   = './save/'
  etf_mem     = 0

  elph        = .true.
  kmaps       = .true.  ! if true, read kmaps from file
  epbread     = .false.
  epwread     = .true.

  lpolar      = .true.
  lifc        = .true.
  use_ws      = .true.
  vme         = .true.

  export_julia = .false.

  scattering  = .true.
  iterative_bte = .true.
  mob_maxiter = 20
  broyden_beta= 1.0
  int_mob     = .false.
  carrier     = .true.
  ncarrier    = -1E15
  mp_mesh_k   = .false.

  efermi_read = .true.
  fermi_energy= 11.4 ! eV
  fsthick     =  0.5 ! eV

  ! Valence band maximum    =  11.477147 eV
  ! Conduction band minimum =  19.670311 eV

  degaussw    = 0.08 ! eV
  nstemp      = 2
  temps       = 200.0 300.0 400.0 ! K

  nkf1        = 15
  nkf2        = 15
  nkf3        = 15

  nqf1        = 15
  nqf2        = 15
  nqf3        = 15

  use_opt_w2b = .true.

  nbndsub     =  8

  wannierize  = .false.
  num_iter    = 1000
  dis_win_max = 100.0
  dis_froz_max= 8.5
  iprint      = 2
  proj(1)     = 'f=-0.125,-0.125,-0.125:sp3:z=0,0,1:x=0,1,0:zona=2.0'
  proj(2)     = 'f= 0.125, 0.125, 0.125:sp3:z=0,0,1:x=1,0,0:zona=2.0'
  wdata(1)    = 'dis_num_iter = 1000'
  wdata(2)    = 'dis_conv_tol = 1E-10'
  wdata(3)    = 'conv_tol = 1E-10'
  wdata(4)    = 'conv_window = 5'

  nk1         = 2
  nk2         = 2
  nk3         = 2

  nq1         = 2
  nq2         = 2
  nq3         = 2
 /
