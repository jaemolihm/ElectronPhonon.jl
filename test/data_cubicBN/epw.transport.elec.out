                                                                                      
                                       ``:oss/                                        
                           `.+s+.     .+ys--yh+     `./ss+.                           
                          -sh//yy+`   +yy   +yy    -+h+-oyy                           
                          -yh- .oyy/.-sh.   .syo-.:sy-  /yh                           
                 `.-.`    `yh+   -oyyyo.     `/syys:    oys      `.`                  
               `/+ssys+-` `sh+      `                   oys`   .:osyo`                
               -yh- ./syyooyo`                          .sys+/oyo--yh/                
               `yy+    .-:-.                             `-/+/:`  -sh-                
                /yh.                                              oys                 
          ``..---hho---------`   .---------..`      `.-----.`    -hd+---.             
       `./osmNMMMMMMMMMMMMMMMs. +NNMMMMMMMMNNmh+.   yNMMMMMNm-  oNMMMMMNmo++:`        
       +sy--/sdMMMhyyyyyyyNMMh- .oyNMMmyyyyyhNMMm+` -yMMMdyyo:` .oyyNMMNhs+syy`       
       -yy/   /MMM+.`-+/``mMMy-   `mMMh:`````.dMMN:` `MMMy-`-dhhy```mMMy:``+hs        
        -yy+` /MMMo:-mMM+`-oo/.    mMMh:     `dMMN/`  dMMm:`dMMMMy..MMMo-.+yo`        
         .sys`/MMMMNNMMMs-         mMMmyooooymMMNo:   oMMM/sMMMMMM++MMN//oh:          
          `sh+/MMMhyyMMMs- `-`     mMMMMMMMMMNmy+-`   -MMMhMMMsmMMmdMMd/yy+           
    `-/+++oyy-/MMM+.`/hh/.`mNm:`   mMMd+/////:-.`      NMMMMMd/:NMMMMMy:/yyo/:.`      
   +os+//:-..-oMMMo:--:::-/MMMo. .-mMMd+---`           hMMMMN+. oMMMMMo. `-+osyso:`   
   syo     `mNMMMMMNNNNNNNNMMMo.oNNMMMMMNNNN:`         +MMMMs:`  dMMMN/`     ``:syo   
   /yh`     :syyyyyyyyyyyyyyyy+.`+syyyyyyyyo:`         .oyys:`   .oyys:`        +yh   
   -yh-        ````````````````    `````````              ``        ``          oys   
   -+h/------------------------::::::::://////++++++++++++++++++++++///////::::/yd:   
   shdddddddddddddddddddddddddddddhhhhhhhhyyyyyssssssssssssssssyyyyyyyhhhhhhhddddh`   
                                                                                      
  S. Ponce, E. R. Margine, C. Verdi, and F. Giustino,                                 
                                                Comput. Phys. Commun. 209, 116 (2016) 
                                                                                      

     Program EPW v.5.3.1 starts on  2Nov2021 at 13:19:53 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Reading input from epw.transport.elec.in

     WARNING: Mismatch between temps(:) and nstemp

     WARNING: Using supplied temperature list and ignoring nstemp
                      
     ------------------------------------------------------------------------ 
                   RESTART - RESTART - RESTART - RESTART                         
     Restart is done without reading PWSCF save file.                  
     Be aware that some consistency checks are therefore not done.                  
     ------------------------------------------------------------------------ 
                      

     --                                                                         

     bravais-lattice index     =            0
     lattice parameter (a_0)   =       0.0000  a.u.
     unit-cell volume          =       0.0000 (a.u.)^3
     number of atoms/cell      =            0
     number of atomic types    =            0
     kinetic-energy cut-off    =       0.0000  Ry
     charge density cut-off    =       0.0000  Ry
     Exchange-correlation= not set
                           (  -1  -1  -1  -1  -1  -1  -1)


     celldm(1)=    0.00000  celldm(2)=    0.00000  celldm(3)=    0.00000
     celldm(4)=    0.00000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of a_0)
               a(1) = (  0.0000  0.0000  0.0000 )  
               a(2) = (  0.0000  0.0000  0.0000 )  
               a(3) = (  0.0000  0.0000  0.0000 )  

     reciprocal axes: (cart. coord. in units 2 pi/a_0)
               b(1) = (  0.0000  0.0000  0.0000 )  
               b(2) = (  0.0000  0.0000  0.0000 )  
               b(3) = (  0.0000  0.0000  0.0000 )  


     Atoms inside the unit cell: 

   Cartesian axes

     site n.  atom      mass           positions (a_0 units)

 
     No symmetry!

     G cutoff =    0.0000  (      0 G-vectors)     FFT grid: (  0,  0,  0)
     number of k points=    0
                       cart. coord. in units 2pi/a_0
     EPW          :      0.00s CPU      0.00s WALL

     EPW          :      0.00s CPU      0.00s WALL


     -------------------------------------------------------------------
     Using bn.ukk from disk
     -------------------------------------------------------------------

     Symmetries of Bravais lattice:  48
     Symmetries of crystal:          24

     Do not need to read .epb files; read .fmt files


     Band disentanglement is used: nbndsub =    8

     Computes the analytic long-range interaction for polar materials [lpolar]
      
     Construct the Wigner-Seitz cell using Wannier centers and atomic positions 
     Number of WS vectors for electrons      123
     Number of WS vectors for phonons       25
     Number of WS vectors for electron-phonon       25
     Maximum number of cores for efficient parallelization       50

     Reading Hamiltonian, Dynamical matrix and EP vertex in Wann rep from file


     Reading interatomic force constants

     Read Z* and epsilon
     IFC last    0.0091940
      Imposed simple ASR

     Finished reading ifcs


     Finished reading Wann rep data from file

     ===================================================================
     Memory usage:  VmHWM =        62Mb
                   VmPeak =      2611Mb
     ===================================================================
     
     Using uniform q-mesh:   15  15  15
     Size of q point mesh for interpolation:       3375
     Using uniform MP k-mesh:   15  15  15
     Size of k point mesh for interpolation:        408
     Max number of k points per pool:              408

     Fermi energy coarse grid =  10.921574 eV

     ===================================================================

     Fermi energy is read from the input file: Ef =  15.400000 eV

     ===================================================================

              ibndmin =     5  ebndmin =    15.371 eV
              ibndmax =     5  ebndmax =    15.750 eV


     Number of ep-matrix elements per pool :         1224 ~=    9.56 Kb (@ 8 bytes/ DP)

     File ./Fepmatkqcb1/bn.epmatkqcb1_0 deleted, as requested
      
     A selecq.fmt file was found but re-created because selecqread == .FALSE. 
     We only need to compute       58 q-points
      
     Valence band maximum    =  10.921574 eV
     Conduction band minimum =  15.371436 eV

     Temperature  200.000 K
     Mobility CB Fermi level =  15.359384 eV

     Temperature  300.000 K
     Mobility CB Fermi level =  15.353358 eV

     Temperature  400.000 K
     Mobility CB Fermi level =  15.347321 eV

     ===================================================================
     Scattering rate for IBTE
     ===================================================================

     Restart and restart_step inputs deactivated (restart point at every q-points).
     No intermediate mobility will be shown.

     Fermi Surface thickness =   0.400000 eV
     This is computed with respect to the fine Fermi level  15.400000 eV
     Only states between  15.000000 eV and  15.800000 eV will be included

     Save matrix elements larger than threshold:   0.877914951989E-23
      
      200.000     15.3594       0.100000E+21
      300.000     15.3534       0.999996E+20
      400.000     15.3473       0.100001E+21
      
     epmatkqread automatically changed to .TRUE. as all scattering have been computed.
      
     ===================================================================
     Memory usage:  VmHWM =        66Mb
                   VmPeak =      2730Mb
     ===================================================================

     Number of elements per core        215
     Symmetry mapping finished
      
     =============================================================================================
     BTE in the self-energy relaxation time approximation (SERTA)
     =============================================================================================

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.484502E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.484502E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.484502E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.313252E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.313252E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.313252E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.224213E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.224213E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.224213E+03
      
     =============================================================================================
     Start solving iterative Boltzmann Transport Equation
     =============================================================================================

     Iteration number:         1

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.313101E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.313101E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.313101E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.199038E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.199038E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.199038E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.141522E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.141522E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.141522E+03

                                                      0.313101E+03    Max error
     Iteration number:         2

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.373751E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.373751E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.373751E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.240845E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.240845E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.240845E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.172506E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.172506E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.172506E+03

                                                      0.606493E+02    Max error
     Iteration number:         3

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.352290E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.352290E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.352290E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.225540E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.225540E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.225540E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.160890E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.160890E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.160890E+03

                                                      0.214607E+02    Max error
     Iteration number:         4

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.359884E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.359884E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.359884E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.231143E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.231143E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.231143E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.165245E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.165245E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.165245E+03

                                                      0.759382E+01    Max error
     Iteration number:         5

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357197E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357197E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357197E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229092E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229092E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229092E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.163612E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.163612E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.163612E+03

                                                      0.268706E+01    Max error
     Iteration number:         6

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.358148E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.358148E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.358148E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229843E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229843E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229843E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164224E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164224E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164224E+03

                                                      0.950814E+00    Max error
     Iteration number:         7

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357811E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357811E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357811E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229568E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229568E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229568E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.163995E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.163995E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.163995E+03

                                                      0.336444E+00    Max error
     Iteration number:         8

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357930E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357930E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357930E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229668E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229668E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229668E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164081E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164081E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164081E+03

                                                      0.119050E+00    Max error
     Iteration number:         9

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357888E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357888E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357888E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229632E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229632E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229632E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164049E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164049E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164049E+03

                                                      0.421258E-01    Max error
     Iteration number:        10

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357903E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357903E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357903E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229645E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229645E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229645E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164061E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164061E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164061E+03

                                                      0.149061E-01    Max error
     Iteration number:        11

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357898E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357898E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357898E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229640E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229640E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229640E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164056E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164056E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164056E+03

                                                      0.527452E-02    Max error
     Iteration number:        12

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357900E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357900E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357900E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229642E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229642E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229642E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164058E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164058E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164058E+03

                                                      0.186638E-02    Max error
     Iteration number:        13

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.661981E-03    Max error
     Iteration number:        14

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.242351E-03    Max error
     Iteration number:        15

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.896416E-04    Max error
     Iteration number:        16

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.336096E-04    Max error
     Iteration number:        17

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.126014E-04    Max error
     Iteration number:        18

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.472468E-05    Max error
     Iteration number:        19

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.177144E-05    Max error
     Iteration number:        20

     =============================================================================================
       Temp     Fermi   Elec density  Population SR                  Elec mobility 
        [K]      [eV]     [cm^-3]      [e per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  15.3594   0.10000E+21   0.00000E+00    0.357899E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.357899E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.357899E+03
      300.000  15.3534   0.10000E+21   0.00000E+00    0.229641E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.229641E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.229641E+03
      400.000  15.3473   0.10000E+21   0.00000E+00    0.164057E+03    0.000000E+00    0.000000E+00
                                                      0.000000E+00    0.164057E+03    0.000000E+00
                                                      0.000000E+00    0.000000E+00    0.164057E+03

                                                      0.664173E-06    Max error

     Unfolding on the coarse grid
     elphon_wrap  :      0.01s CPU      0.01s WALL (       1 calls)

     INITIALIZATION: 




     Electron-Phonon interpolation
     ephwann      :      7.05s CPU      7.13s WALL (       1 calls)
     ep-interp    :      3.75s CPU      3.81s WALL (      58 calls)

     DynW2B       :      0.01s CPU      0.01s WALL (      58 calls)
     HamW2B       :      0.84s CPU      0.86s WALL (   24276 calls)
     ephW2Bp      :      0.89s CPU      0.90s WALL (      58 calls)
     ephW2B       :      0.01s CPU      0.01s WALL (      72 calls)
     print_ibte   :      0.02s CPU      0.02s WALL (      58 calls)
     vmewan2bloch :      0.02s CPU      0.02s WALL (     144 calls)
     vmewan2bloch :      0.02s CPU      0.02s WALL (     144 calls)


     Total program execution
     EPW          :      7.06s CPU      7.17s WALL

     ===============================================================================
     The functionality-dependent EPW.bib file was created with suggested citations. 
     Please consider citing the papers listed in EPW.bib.                           
     ===============================================================================
                                                                                    
